linkerd/app/inbound/src/metrics/error/http.rs

```diff
@@ -0,0 +1,84 @@
+use super::ErrorKind;
+use linkerd_app_core::{
+    metrics::{metrics, Counter, FmtMetrics, ServerLabel},
+    svc::{self, stack::NewMonitor},
+    transport::{labels::TargetAddr, OrigDstAddr},
+    Error,
+};
+use parking_lot::RwLock;
+use std::{collections::HashMap, sync::Arc};
+
+metrics! {
+    inbound_http_errors_total: Counter {
+        "The total number of inbound HTTP requests that could not be processed due to a proxy error."
+    }
+}
+
+#[derive(Clone, Debug, Default)]
+pub struct HttpErrorMetrics(Arc<RwLock<HashMap<(ErrorKind, (TargetAddr, ServerLabel)), Counter>>>);
```

Comment:
similarly, it looks like there are never concurrent reads of the `RwLock` so it should probably just be a `Mutex`?